### DISCUSSION TITLE
How isn't there any model for me to choice once I logon openwebui?

### DISCUSSION THREAD

**Discussion by amdhugo** [open-webui/open-webui#3789] [Q&A] [answered]
I setup ollama and open-webui on same computer that is running ubuntu. ollama is able to run well locally in a terminal. I can logon open-webui as well. I refer to the page to launch open-webui in a docker. https://docs.openwebui.com/. However, even I tried command either shows at 'If Ollama is on your computer' or 'For CPU Only', there is empty in model select drop list. I guess it might be environment issue but I don't know where the issue is. Can anyone who can help? 
I tied those two commands:
docker run -d -p 3000:8080 -v /usr/share/ollama/.ollama:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
docker run -d -p 3000:8080 -v ollama:/usr/share/ollama/.ollama -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:ollama

**Comment by amdhugo** [comment]
attach screen capture,
![Screenshot 2024-07-11 160825](https://github.com/open-webui/open-webui/assets/64545075/b51c3650-5143-4d49-b86e-9883713e158a)

**Comment by SlOrbA** [comment, marked as answer]
I've been running with the following volume mappings "-v ollama:/root/.ollama -v open-webui:/app/backend/data" and with the :ollama  tag. Local models naturally com from the Ollama and it can be that you haven't yet loaded any models. In the lates version I think you can load models under the Admin settings and models section.

**Reply by amdhugo** [reply]
I tried to use dev branch like this 
docker run -d -p 3000:8080  -v ollama:/usr/share/ollama/.ollama -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:dev
After then, how to load models, like lama3? Is there a guide? I click the download button of 'pull a model from ollama.com' but I don't know the name to be entered :(
![image](https://github.com/user-attachments/assets/89a6ae36-1a25-4ce0-b49b-68ae76089ebe)

**Reply by amdhugo** [reply]
What I see once I input 'llama3:8B' and click download, I always get prompt 'download cancelled'. Not sure what is going on.
